src/Microsoft.DotNet.Darc/DarcLib/Remote.cs

```diff
@@ -228,7 +228,7 @@ public async Task<List<GitFile>> CommitUpdatesAsync(
             {
                 engCommonFiles = engCommonFiles
                     .Select(f => new GitFile(
-                        f.FilePath.Replace(VmrInfo.ArcadeRepoDir, null),
+                        f.FilePath.Replace(new UnixPath(VmrInfo.ArcadeRepoDir), null).TrimStart('/'),
```

Comment:
I think `ArcadeRepoDir` should be a `UnixPath` instead

Reply:
Or maybe not really

Reply:
Depends how it's used in most scenarios - for getting a file from remote repo (where I see this), it's possible easier to have it a `UnixPath`. Then when working with git index, it also expects usually `/` and only local file access is suitable for `NativePath`. But usually local file access is prepended by the repository path, which is an absolute native path which would turn whatever comes after to native path.

So I think we should make `ArcadeRepoDir` a `UnixPath`

Reply:
Yeah I had a look a the code, I think it's mostly used like you said, and is probably better to keep this way